components/claim-mgt/org.wso2.carbon.claim.mgt.ui/src/main/resources/web/claim-mgt/claim-view.jsp

```diff
@@ -222,11 +223,11 @@
                 if (claims[j].getClaim().getDisplayTag() != null) {%>
             <div class="trigger-title-container">
                 <a href="#" class="trigger-title"><i class="claim-arrow-down"></i>
-                    <span><%=claims[j].getClaim().getDisplayTag()%></span></a>
-                <a href="update-claim.jsp?dialect=<%=dialectUri%>&claimUri=<%=claims[j].getClaim().getClaimUri()%>"
+                    <span><%=Encode.forHtmlContent(claims[j].getClaim().getDisplayTag())%></span></a>
+                <a href="update-claim.jsp?dialect=<%=Encode.forUriComponent(dialectUri)%>&claimUri=<%=Encode.forUriComponent(claims[j].getClaim().getClaimUri())%>"
                    class="editLink icon-link">Edit</a>
                 <a href="#" class="icon-link deleteLink" style="background-image:url(../claim-mgt/images/delete.gif);"
-                   onclick="removeItem('<%=dialectUri%>','<%=claims[j].getClaim().getClaimUri()%>','<%=claims.length%>'  );return false;"><fmt:message
+                   onclick="removeItem('<%=Encode.forHtmlAttribute(dialectUri)%>','<%=Encode.forHtmlAttribute(claims[j].getClaim().getClaimUri())%>','<%=Encode.forHtmlAttribute(String.valueOf(claims.length))%>'  );return false;"><fmt:message

```

Comment:
This is javascript execution context. So you have to encode for javascript attribute here. We cannot encode for html attribute since the passed parameters are used in an html content and as well as in a query string